phpcs.xml

```diff
@@ -1,23 +0,0 @@
-<?xml version="1.0"?>
```

Comment:
Are we ditching this?

Reply:
only if I kept `phpcs --ignore=vendor,web --extensions=php --standard=PSR12 --exclude=PSR1.Files.SideEffects .` which p much does the same thing